packages/synapse-react-client/src/components/SynapseTable/SynapseTable.integration.test.tsx

```diff
@@ -0,0 +1,611 @@
+import { render, screen, waitFor } from '@testing-library/react'
```

Comment:
New integration test suite mostly based on old tests. There's still probably a few cases to backfill